packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -329,6 +329,40 @@ export const components = (
         }),
       },
     },
+    MuiBreadcrumbs: {
+      styleOverrides: {
+        li: {
+          fontSize: odysseyTokens.TypographySizeBody,
+          lineHeight: odysseyTokens.TypographyLineHeightUi,
+
+          "& a, & button": {
+            borderRadius: odysseyTokens.BorderRadiusTight,
+            color: odysseyTokens.TypographyColorSubordinate,
+            display: "flex",
+            gap: odysseyTokens.Spacing1,
+            padding: odysseyTokens.Spacing1,
+            transitionProperty: "color, background-color",
+            transitionDuration: "100ms",
+            transitionTimingFunction: "linear",
+
```

Comment:
Can we do `& > a`? Or is it that we don't know what they are?

Also, I bet there's more in the `styleOverrides` we could do. Instead of `"& a"`, why not use one of the identifiers in `styleOverrides`? Like normally there's `root`, and you have `li` here. I'm wondering if the other two are also able to be styled more specifically like that.

Reply:
This is specifically targeting the `<Link>` or `<Button>` items that are rendered within the `BreadcrumbList`. Links and buttons need to look the same in this context, but we don't have direct access to their class names because they're not part of the `MuiBreadcrumbs` itself.